### DISCUSSION TITLE
Tell me your favorite programming language

### DISCUSSION THREAD

**Discussion by ou-junya** [NAIST-SE/AttractingContributors#1036] [2024 Attraction Contributions]
https://github.com/ou-junya/FavoriteProgrammingLanguage

I'd like to know what your favorite programming language is.
Please tell me your favorite programming language.

**Comment by Samabena** [comment]
Hello @ou-junya , I recently made a contribution to your project , could you please contribute to my as well ? Following is the link to [my repository](https://github.com/Samabena/sighting ) . Thanks in advance.

**Comment by mustafizur-r** [comment]
Hello [@ou-junya](https://github.com/ou-junya) , I recently made a contribution to your project , I have added “C#" could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.
